data/uber.txt

```diff
@@ -331,6 +399,48 @@ Jolly Nature
 - Stone Edge
 - Iron Tail
 
+
+=== [ubers] Latios ===
+
+Latios @ Soul Dew
+Ability: Levitate
+EVs: 4 Def / 252 SpA / 252 Spe
+Timid Nature
+- Draco Meteor
+- Psyshock
+- Defog
+- Hidden Power [Fire] / Memento / Recover
+
+Latios @ Soul Dew
+Ability: Levitate
+EVs: 4 Def / 252 SpA / 252 Spe
+Timid Nature
+- Draco Meteor
+- Psyshock
+- Calm Mind
```

Comment:
Currently, yes. However, since there is specific code in the team generator that handles Defog, Stealth Rock,and friends, I'd like them to be kept separated.